spannerlib/wrappers/spannerlib-dotnet/spannerlib-dotnet-grpc-tests/spannerlib-dotnet-grpc-tests.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net9.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
We target .NET 8 as the minimum. The `<RollForward>Major</RollForward>` tag below tells .NET that it is OK to run these tests on higher major .NET versions (e.g. it is OK to run the tests on .NET 9)